build.sbt

```diff
@@ -2,20 +2,22 @@ name := "millfork"
 
 version := "0.3.23-SNAPSHOT"
 
-scalaVersion := "2.12.11"
+scalaVersion := "2.12.12"
```

Comment:
I haven't gotten Scala 2.12.12 to work with GraalVM yet. Unless 2.12.12 is necessary, I'd keep it at .11.

Reply:
Reverted